resident/resident-service/src/main/java/io/mosip/resident/service/impl/DownloadCardServiceImpl.java

```diff
@@ -186,6 +186,7 @@ private ResponseWrapper<CheckStatusResponseDTO> getCheckStatusResponse(HashMap<S
         checkStatusResponseDTOResponseWrapper.setResponse(checkStatusResponseDTO);
         checkStatusResponseDTOResponseWrapper.setId(this.environment.getProperty(ResidentConstants.CHECK_STATUS_INDIVIDUAL_ID));
         checkStatusResponseDTOResponseWrapper.setVersion(this.environment.getProperty(ResidentConstants.CHECKSTATUS_INDIVIDUALID_VERSION));
+        checkStatusResponseDTOResponseWrapper.setResponsetime(DateUtils.getUTCCurrentDateTimeString());
```

Comment:
Replace the value with `DateUtils.formatToISOString(DateUtils.getUTCCurrentDateTime())`

Reply:
Updated changes